pages/linux/users.md

```diff
@@ -0,0 +1,11 @@
+# users
+
+> Display a list of logged in users.
+
+- Display a list of logged in users:
+
+`users`
+
+- Display a list of logged in users according to a specific file:
```

Comment:
What does "according to a file" mean ? What does the file specify ?

Reply:
You can specify a custom file. By default it checks `/var/run/utmp`. But there's also `/var/log/wmtp` which is a historical dump of `utmp`. And `btmp` which displays failed login attempts.

Reply:
Ah I see. I thought it will always show the current logged in users. Perhaps we can mention "/var/log/wmtp" instead of path/to/file ? That may make things more clear.